src/controllers/LoginController.ts

```diff
@@ -36,11 +42,30 @@ export default class LoginController {
     if (!isPasswordValid) {
       return res.status(401).json({ message: 'Invalid email or password' })
     }
-
+    LoginController.loggedInUserId = existingUser.id
     const token = generateToken({
       id: existingUser.id,
       email: existingUser.email,
     })
+    redisClient.setEx(`user:${existingUser.id}`, 24 * 60 * 60, token)
     return res.status(200).json({ jwt: token, message: 'Login successful' })
   }
+
+  // Handles user logout
+  static async logOut(
+    req: AuthenticatedRequest,
+    res: Response,
+  ): Promise<Response> {
+    const userId = LoginController.loggedInUserId
```

Comment:
```suggestion
  const loggedInUser=req.user
```

Reply:
please make sure that you using isAuthenticated middleware, and and using req.user to get authenticated user
for reference you can checkout this PR #56